src/scripts/main.js

```diff
@@ -1 +1,28 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+// const form = document.querySelector('.subscribe__form');
+// const inputs = document.querySelectorAll('.subscribe__input');
+
+// form.addEventListener('submit', function(event) {
+//   event.preventDefault();
+
+//   inputs.forEach(function(elem)
+//   {
+//     elem.value = '';
+//   });
+// });
+
+const form = document.querySelector('subscribe__form');
```

Comment:
this should be the name of the class. And move submit button inside the form
![image](https://user-images.githubusercontent.com/99403198/201654086-8cdf8d14-1938-4a87-81e4-4ea724d6186d.png)

```suggestion
const form = document.querySelector('.subscribe__form');
```